k8s/client.go

```diff
@@ -58,15 +60,17 @@ func GetKubeAPIServerVersion() (*version.Info, error) {
 }
 
 // IsOpenShift - Returns a boolean which indicates if we are running in an OpenShift cluster
-func IsOpenShift() (bool, error) {
+func IsOpenShift(log *zap.SugaredLogger) (bool, error) {
 	k8sClientSet, err := GetClientSetWrapper()
 	if err != nil {
 		return false, err
 	}
 
 	serverGroups, _, err := k8sClientSet.Discovery().ServerGroupsAndResources()
```

Comment:
should we add a check for cases where serverGroups is nil?

Reply:
yes, done.